src/main/java/org/thoughtslive/jenkins/plugins/jira/service/JiraService.java

```diff
@@ -38,11 +40,28 @@ public JiraService(final Site jiraSite) {
 
     final ConnectionPool CONNECTION_POOL = new ConnectionPool(5, 60, TimeUnit.SECONDS);
 
-    OkHttpClient httpClient = new OkHttpClient.Builder()
-        .connectTimeout(jiraSite.getTimeout(), TimeUnit.MILLISECONDS)
-        .readTimeout(jiraSite.getReadTimeout(), TimeUnit.MILLISECONDS)
-        .connectionPool(CONNECTION_POOL)
-        .retryOnConnectionFailure(true).addInterceptor(new SigningInterceptor(jiraSite)).build();
+    OkHttpClient.Builder OkHttpClientBuilder = new OkHttpClient.Builder()
```

Comment:
nit: Can we rename the variable to follow the java naming conventions, just lowercase O